CHANGES.md

```diff
@@ -1,3 +1,10 @@
+# 22.02
+
+   * Microphysics has moved from Starkiller-Astro to AMReX-Astro.  The
+     git submodules have been updated accordingly.  The old URL should
+     redirect to the new location, but you are encouraged to change
+     the submodule URL if you use submodules.
```

Comment:
Might want to explicitly note the new URL (or even provide the command to do the submodule move).

Reply:
done